cmd/horcrux/cmd/cosigner.go

```diff
@@ -45,7 +55,19 @@ func StartCosignerCmd() *cobra.Command {
 				pv       types.PrivValidator
 				chainID  = config.ChainID
 				logger   = tmlog.NewTMLogger(tmlog.NewSyncWriter(os.Stdout)).With("module", "validator")
-				config   = signer.Config{
+				cfg      signer.Config
+			)
+
+			if single {
```

Comment:
Thank you for catching this bone headed move. One easy refactor will be to remove the `signer.Config` object from the `signer/` lib and use the one here instead